docs/index.md

```diff
@@ -52,3 +22,14 @@ We're also happy to help if you create an issue to our
 important is probably to explain what happened with which version of rdiff-backup,
 with which command parameters on which operating system version, and attach the output
 of rdiff-backup run with the very verbose option `-v9`.
+
+This is an open source project and contributions are welcome!
+
+
+## History
+
+Rdiff-backup has been around for almost 20 years now and has proved to be a very solid solution for backups and it is still unique in its model of unlimited incrementals with no need to space consuming regular full backups.
+
+Current lead developers are Eric Lavar, Patric Dufresene and Otto Kekäläinen. Full list of core developers available at [rdiff-backup Github page](https://github.com/rdiff-backup/rdiff-backup/people) and on the [credits page](credits.md).
+
+The original author and maintainer was **Ben Escoto** from 2001 to 2005. Key contributors from 2005 to 2016 were Dean Gaudet, Andrew Ferguson and Edward Ned Harvey. After some hibernation time Sol1 took over the stewardship of rdiff-backup from February 2016 but there were no new releases. In August 2019 [Eric Lavar](https://www.lavar.de/) with the support of Otto Kekäläinen from [Seravo](https://seravo.com/) and Patrik Dufresne from [Miranca](http://www.patrikdufresne.com/en/minarca/) took over, completed the Python 3 rewrite and finally released rdiff-backup 2.0 in March 2020.
```

Comment:
It's Lavarde, the domain is a pun. And I think "Minarca" is correct.

Reply:
Thanks, I was a bit unsure when all your commits say "Eric L."